buildpipeline/Core-Setup-Sign-And-Publish.json

```diff
@@ -45,7 +45,7 @@
     },
     {
       "environment": {},
-      "enabled": true,
```

Comment:
I disabled the steps to download the nuget tooling, copy the signed packages, and publish to myget because the "download tooling" step fails - I thought the script was a machine script, but it's actually in the PipeBuild repo. I'll have to do what CoreClr does in order to get this working (Clone PipeBuild, Clone Core-Setup, run the powershell script, then work out of the Core-Setup enlistment)